### PR TITLE
Check that input files are OpenAPI 3.0.x

### PR DESCRIPTION
**Pull request by hudlow**
## PR summary
The validator does not provide useful output for Swagger 2 or OpenAPI 3.1.x files, so throw an error if an input file is not OpenAPI 3.0.x.

### General checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- ~Dependencies have been updated as needed~
- ~.secrets.baseline updated as needed?~

### CODE REVIEW

File: packages/validator/test/cli-validator/mock-files/openapi-3-1.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "swagger": "2.0" }
```
Comment (padamstx) [inline packages/validator/test/cli-validator/mock-files/openapi-3-1.json:1]: Shouldn't this one contain openapi 3.1?

File: packages/validator/test/cli-validator/mock-files/swagger-2.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{ "openapi": "3.1.0" }
```
Comment (padamstx) [inline packages/validator/test/cli-validator/mock-files/swagger-2.json:1]: shouldn't this one contain swagger 2?

**Review by padamstx — CHANGES_REQUESTED** [review]
Looks good but it looks like you inadvertently swapped the two new test files :)

File: packages/validator/test/cli-validator/mock-files/openapi-3-1.json (hunk 1 above)
Comment (dpopp07) [reply packages/validator/test/cli-validator/mock-files/openapi-3-1.json:1]: Haha, can't believe I didn't catch this!

Comment (hudlow) [reply packages/validator/test/cli-validator/mock-files/openapi-3-1.json:1]: Hah. Oh man, when I duplicate a file, my editor sometimes confuses me on which on I'm in.

Comment (padamstx) [reply packages/validator/test/cli-validator/mock-files/openapi-3-1.json]: haha, might be time to switch to a different editor :joy:
I hear that emacs is available LMAO

**Review by padamstx — APPROVED** [review]
LGTM!

**Pull request merged**